### ISSUE TITLE
[Feature] Integration with Animated for smooth animations

### ISSUE THREAD

**Issue by nikgraf** [HippoAR/react-native-arkit#69]
This would be great to move elements. Maybe there is already a way to animated, but I'm not aware of it.

**Comment by macrozone** [comment]
i could manage to move objects by updating the positions in a requestAnimationFrame loop.

But it would probably perform better if we could leaverage scenekits native animation api.

**Comment by nikgraf** [comment]
yeah, did the same, but with a couple of objects it go quite laggy

**Comment by macrozone** [comment]
@nikgraf if you have multiple objects and move only one around, make sure, that your other objects are not updated unnecessary. 

You can use recompose' pure or onlyupdateforkeys hoc to achieve that: https://github.com/acdlite/recompose/blob/master/docs/API.md

Having said that, this optimisations should be done directly in the library in the future.

**Comment by macrozone** [comment]
Update: the current version on github has an improvement on the render performance when moving stuff around. It should also reduce unnecessary updates. I still recommend, that you try to control updates yourself with pure or other performance-hocs from recompose or componentShouldUpdate on component classes.

Also you can specify basic transition, which just smooths every change in position property. (see https://github.com/HippoAR/react-native-arkit/pull/76)

i will release it after some testing this evening, but you can use it right now:

`
  "react-native-arkit":
      "https://github.com/HippoAR/react-native-arkit.git#28634fa44ebebf09a30d44ae71a148fc8f2f810a",
`

**Comment by nikgraf** [comment]
@macrozone transitions are great! thanks for all the hard work you put into this

**Comment by nikgraf** [comment]
@macrozone @qftgtr would you accept a PR adding transitions to the docs?

**Comment by macrozone** [comment]
@nikgraf it would be an honor :-)

**Comment by nikgraf** [comment]
will do that later today or tomorrow 🙌
